modules/dag-l1/src/main/resources/db.migration/V1__AddressTable.sql

```diff
@@ -0,0 +1,5 @@
+create table if not exists Address (
+    address varchar(40) primary key,
+    balance bigint default 0
```

Comment:
on L1 we would need also a majority balance

Reply:
Right. But I guess we should add needed pieces the moment they are needed, so here that would be added with majority snapshot processing I think.

Reply:
I'm fine with it :+1: